docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,364 @@
+---
+
+In this guide, you will learn how to create host certificates through Machine ID to allow Teleport users (manually or automated through Machine ID) to access OpenSSH servers. This allows for the benefits of
+Machine ID with OpenSSH nodes and reduces risk by ensuring that short-lived host certificates on Teleport clusters can be successfully applied.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!/docs/pages/includes/tctl.mdx!)
+
+- OpenSSH server `sshd` version 6.9 or above installed. The SSH port on this host must be open to traffic from the Teleport Proxy Service host.
+
+  ```code
+  $ ssh -V
+  ```
+
+## Step 1/4. Download and install Teleport (=teleport.version=)
+
+In this step, you will be downloading and installing Teleport binaries onto the
+machine you wish to assign an identity to.
+
+Each Teleport package hosted on our downloads page ships with several useful
+binaries, including `teleport`, `tctl`, `tsh`, and `tbot`:
+
+- `teleport` is the daemon used to initialize a Teleport cluster; this binary is not used in this guide.
+- `tctl` is the administrative tool you will use to create the bot user.
+- `tsh` is the client tool you will use to log in to the OpenSSH server.
+- `tbot` is the Machine ID tool you will use to associate a bot user with a machine.
+
+Machine ID is available starting from the Teleport 9.0.0 release. Download the appropriate Teleport package for your platform from our [downloads page](https://goteleport.com/teleport/download).
+
+Next, log in to your cluster using `tsh` to ensure that your machine will be able to interact with the cluster:
+
+```code
+tsh login --proxy=myteleportcluster.com --user=teleport-username
+```
+
+
+## Step 2/4. Create a Role to Issue `tbot` Certificates
+
+In order to create certificates `tbot` a role that provides the necessary permissions. Roles may restrict which **principals** can be requested using **predicates**,
+or strings which are used to reduce the scope of permissions for creating host certificates.
+By using the format of `where: 'predicate(host_cert.principals, "domain.name")'`, a Teleport user can create a role that will only be able to able to affect the designated domain name
+in a manner defined by the selected predicate. A list of supported predicates and their behavior are listed below:
+
+| **Predicate** | **Behavior** | **Example** |
+| ----------- | ----------- | ----------- |
+| all_equal | All created domains must be equal to the domain. | `all_equal(host_cert.principals, "foo.example.com")` |
+| all_end_with | All created domains or subdomains must end with the selected domain. | `all_end_with(host_cert.principals, ".example.com")` |
+| is_subset | All created domains must be a subset of the selected domain. This requires the certificate requested by tbot to be be listed with the role's predicate. | `is_subset(host_cert.principals, "foo.example.com", "bar.example.com")` |
```

Comment:
I'd argue this could still be clarified since unlike `all_end_with()`, `is_subset()` doesn't look at domains, just set membership:
```suggestion
| is_subset | All requested principals must be contained in the listed set. | `is_subset(host_cert.principals, "foo.example.com", "bar.example.com")` |
```

I'm also not sure if the "All created domains ..." verbiage is super clear, since we're not creating domains but instead requesting certs with some set of principals.